app/experimenter/nimbus-ui/package.json

```diff
@@ -18,7 +18,9 @@
   },
   "jest": {
     "collectCoverageFrom": [
-      "!src/**/*.stories.*"
+      "!src/**/*.stories.*",
+      "!src/lib/mocks.tsx",
```

Comment:
Ooh, I filed a ticket where I mention this here: https://github.com/mozilla/experimenter/issues/3808

We can discuss later today maybe on if we want to exclude these from coverage?